src/java/htsjdk/tribble/bed/SimpleBEDFeature.java

```diff
@@ -136,4 +137,9 @@ public void setLink(String link) {
     public java.util.List<FullBEDFeature.Exon> getExons() {
         return emptyExonList;
     }
+
+    @Override
```

Comment:
I think that internally we should always convert to 1-based closed, and only convert to other representations when writing formats that require it (like BED files).